projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.scss

```diff
@@ -9,6 +12,17 @@
   background: #293e50;
```

Comment:
No all colors are taken from variables.
Variables should be imported from `projects/step-core/styles/core-variables`
For example check
`step-frontend\projects\step-core\src\lib\components\restore-dialog\restore-dialog.component.scss`